pkg/profilecreator/profilecreator.go

```diff
@@ -309,6 +310,18 @@ func (ghwHandler GHWHandler) GetReservedAndIsolatedCPUs(reservedCPUCount int, sp
 		topologyInfo = ghwHandler.TopologyHTDisabled(topologyInfo)
 
 	}
+	log.Infof("NUMA cell(s): %d", len(topologyInfo.Nodes))
+	totalCPUs := 0
+	for id, node := range topologyInfo.Nodes {
+		coreList := []int{}
+		for _, core := range node.Cores {
+			coreList = append(coreList, core.LogicalProcessors...)
+		}
+		log.Infof("NUMA cell %d : %v", id, coreList)
```

Comment:
why we cannot just use `core.LogicalProcessoers`?

Reply:
You mean why we are using `core.LogicalProcessors...` here?

Reply:
Nevermind, got it. We want all the logical processors of all the cores in one go.